cypress/utils/helpers.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const apiRequest = (url, method = 'POST', body = undefined, qs = undefined) =>
```

Comment:
we should make this an object after url and method so it wont get so difficutl to update if we need to add more